docs/manuals/source/_extensions/bareos-ext.py

```diff
@@ -31,6 +31,7 @@
 from docutils import nodes
 from pprint import pformat
 import re
+from bareos_urls import BareosUrls
```

Comment:
```suggestion
from bareos_urls import BareosUrls
import re
```
keep `from ... import ...` grouped?

Reply:
Yes, that is the idea.

Reply:
I chosed it this way to first import the Python general modules and than continue with the project specific modul.

Reply:
ok.